inside.go

```diff
@@ -44,7 +44,10 @@ func (f *Interface) consumeInsidePacket(packet []byte, fwPacket *firewall.Packet
 		return
 	}
 
-	hostinfo := f.getOrHandshake(fwPacket.RemoteIP)
+	hostinfo, ready := f.getOrHandshake(fwPacket.RemoteIP, func(h *HostInfo) {
+		h.unlockedCachePacket(f.l, header.Message, 0, packet, f.sendMessageNow, f.cachedPacketMetrics)
+	})
```

Comment:
creating this closure for every packet on the hot patch shouldn't be an issue right? Hopefully it doesn't create garbage, at least if not used?

Reply:
In a further branch I have a handshake benchmark and pprof does not show any allocations here.